src/Microsoft.Identity.Client/Features/PublicClientWithTokenCache/TokenCacheAccessor.cs

```diff
@@ -188,6 +189,12 @@ public void SetKeychainSecurityGroup(string keychainSecurityGroup)
             throw new System.NotImplementedException();
         }
 
+        [Obsolete("Use iOSKeychainSecurityGroup instead (See https://aka.ms/msal-net-ios-keychain-security-group)", false)]
```

Comment:
>Obsolete [](start = 9, length = 8)

this one should not be marked as obsolete. it's the new one. Perhaps the one above should be marked as obsolete